databricks/koalas/tests/test_dataframe.py

```diff
@@ -2396,6 +2396,15 @@ def test_reindex(self):
             kdf.reindex(columns=["numbers", "2", "3"], fill_value=0.0).sort_index(),
         )
 
+        index2 = ["A", "C", "D", "E", "0"]
```

Comment:
Oh, could add test with named index?

like

```python
index2 = pd.Index(["A", "C", "D", "E", "0"], name="index")
```

I think maybe It wouldn't work ?

Reply:
Oh yeah, good point.